### ISSUE TITLE
the num_labels in bert_config dont work?

### ISSUE THREAD

**Issue by tongbc** [kpe/bert-for-tf2#40]
the num_labels in bert_config dont work?i want to make a 27 classes classifier,but i add 
  "num_labels": 28 to bert_config , it wont work,still came out with error 
Received a label value of 26 which is outside the valid range of [0, 2).  Label val....

**Comment by kpe** [comment]
@tongbc - there is no `num_labels` config property in bert-for-tf2, i.e. the classification layer is not part of `BertModelLayer`. To do classification just take the `[CLS]` activation (using a lambda layer) and pass it through a dense layer to obtain as many logits (num_labels) as needed. Also check the `examples/`.
